lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -9,7 +9,7 @@
   <%- end -%>
   </div>
 
-  <div class="form-actions">
+  <div class="form-actions btn btn-dark">
```

Comment:
Out of curiosity, what exactly does this change? I see it's somehow connected to simple form?